### PR TITLE
Share button

### PR DESCRIPTION
**Pull request by joshreisner**
@enderv here's a branch that is a start for how this might work - do you have any thoughts / feedback so far?

not sure if i'm detecting sharing capability in the right way, also how prevalent the `url` property is in people's data

Screenshot 2023-01-26 at 8 41 51 AM

### CODE REVIEW

File: src/helpers/data/load-meeting-data.ts [hunk 1]
```diff
@@ -502,6 +509,10 @@ export function loadMeetingData(
     (window.location.protocol === 'https:' ||
       window.location.hostname === 'localhost');
 
+  //determine sharing
+  capabilities.sharing =
+    typeof navigator.canShare === 'function' && navigator.canShare();
```
Comment (enderv) [inline src/helpers/data/load-meeting-data.ts:514]: LGTM

File: src/components/Meeting.tsx [hunk 2]
```diff
@@ -356,6 +356,18 @@ export default function Meeting({
                       )}
                     </div>
                   )}
+                  {meeting.url && state.capabilities.sharing && (
+                    <Button
+                      onClick={() =>
+                        navigator.share({
+                          title: meeting.name,
+                          url: meeting.url,
```
[the remark this answers is not in the document]
Comment (joshreisner) [reply src/components/Meeting.tsx:364]: what about if it's a react native app (i thought this was your use-case) -- wouldn't that be `localhost:3000`?

**Comment by enderv** [comment]
yep on the use case, but ran into some other issues with the native
approach so current plan is just webview to a hosted site (also helps with
app maintenance, upkeep, etc) so don't need to really worry about the
localhost thing from my side.

**Comment by joshreisner** [comment]
ok @enderv now it looks for `meeting.url` but if it doesnt find it it will fall back to `location.href`

give it a 👍 if you think it's good to go.

**Review by enderv — APPROVED** [review]
LGTM

**Pull request merged**